docs/reference/search/search-your-data/semantic-search-deploy-model.asciidoc

```diff
@@ -0,0 +1,92 @@
+[[semantic-search-deployed-nlp-model]]
+=== Tutorial: semantic search with a deployed model
+
+++++
+<titleabbrev>Semantic search with deployed model</titleabbrev>
+++++
+
+IMPORTANT: For the easiest way to perform semantic search in the {stack}, refer to the <<semantic-search-semantic-text, `semantic_text`>> end-to-end tutorial.
+
+This guide shows you how to implement semantic search with models deployed in {es}: from selecting an NLP model, to writing queries.
```

Comment:
Maybe could have a couple of background sentences that explain that this tutorial was written before the inference endpoints and semantic_text options were introduced and therefore are a little dated. Just one or two extra lines of context might be helpful to re-route the reader.

Maybe that could all live in the IMPORTANT block too?